pom.xml

```diff
@@ -32,35 +62,27 @@
 		<dependency>
 			<groupId>org.jline</groupId>
 			<artifactId>jline</artifactId>
```

Comment:
true, specially useful once we divide core and api (https://github.com/ericbottard/spring-shell2/issues/43 and https://github.com/ericbottard/spring-shell2/issues/44)